src/views/Accessibility/AccessibilityRequestDetailPageRTL.test.tsx

```diff
@@ -162,5 +162,161 @@ describe('AccessibilityRequestDetailPage', () => {
       const [, notesList] = getAllByRole('list');
       expect(within(notesList).getAllByRole('listitem').length).toEqual(2);
     });
+
+    describe('Add a note', () => {
+      it('shows a success message', async () => {
+        const createNoteMocks = [
+          testerMocks[0],
+          {
+            request: {
+              query: CreateAccessibilityRequestNote,
+              variables: {
+                requestID: 'a11yRequest123',
+                note: 'I am an important note',
+                shouldSendEmail: false
+              }
+            },
+            result: {
+              data: {
+                createAccessibilityRequestNote: {
+                  id: 'noteID2',
+                  createdAt: 'time',
+                  authorName: 'Common Human',
+                  note: 'This is such a success'
+                }
+              }
+            }
+          }
+        ];
+
+        const providers = buildProviders(
+          createNoteMocks,
+          testerStore,
+          <AccessibilityRequestDetailPage />
+        );
+        const { findByRole, getByRole } = render(providers);
+        expect(await findByRole('alert')).toBeInTheDocument();
+        const { getByText } = within(getByRole('alert'));
+        expect(await getByText('Success')).toBeInTheDocument();
+        expect(
+          await getByText('Note added to My Special Request')
+        ).toBeInTheDocument();
+      });
+
+      it('can navigate through note tab', async () => {
```

Comment:
The bulk of this test seems unnecessary to me. We shouldn't need to test the all the keyboard navigation for the Tabs. That component is already thoroughly tested.